controller/user/user.go

```diff
@@ -77,11 +78,38 @@ func CreateUser(w http.ResponseWriter, r *http.Request) {
 	_, _ = w.Write(data)
 }
 
+func ChangeName(w http.ResponseWriter, r *http.Request) {
+	token := r.Header.Get("x-token")
+
+	body := r.Body
+	defer body.Close()
+
+	buf := new(bytes.Buffer)
+	if _, err := io.Copy(buf, body); err != nil {
+		errorResponse(err, w)
+	}
+
+	var reqBody request.UpdateRequest
+	if err := json.Unmarshal(buf.Bytes(), &reqBody); err != nil {
+		errorResponse(err, w)
+	}
+
+	var u user.UserData
+	if err := u.UpdateName(reqBody.Name, token); err != nil {
```

Comment:
ここは引数で与えてもいいし、下記のようにUserDataにマッピングしてあげても良いよ！
```suggestion
	var u user.UserData
	u.Name = reqBody.Name
	if err := u.UpdateName(token); err != nil {
```

Reply:
修正しました！